numpy/core/src/umath/loops_umath_fp.dispatch.c.src

```diff
@@ -1,14 +1,30 @@
 /*@targets
- ** $maxopt baseline avx512_skx
+ ** $maxopt baseline
+ ** avx512_skx
+ ** asimd
  */
+/*
+ * Copyright 2023 Arm Limited and/or its affiliates <open-source-office@arm.com>
+ */
+
 #include "numpy/npy_math.h"
 #include "simd/simd.h"
 #include "loops_utils.h"
 #include "loops.h"
 #include "npy_svml.h"
 #include "fast_loop_macros.h"
 
-#if NPY_SIMD && defined(NPY_HAVE_AVX512_SKX) && defined(NPY_CAN_LINK_SVML)
+#if NPY_SIMD
+#if defined(NPY_HAVE_ASIMD) && defined(NPY_CAN_LINK_AOR)
```

Comment:
Same here, I think ADVSIMD is preferable.

Reply:
See previous comment.